Rollup Build Fails Due to Missing `tslib`

### Bug Report: Rollup Build Fails Due to Missing `tslib`

**Description**  
Rollup fails to build the project when encountering certain syntax that requires `tslib`. The error indicates that the `tslib` module cannot be found, leading to a failed build process.

**Error Output**  
```
rollup v3.29.4 bundles .\\/src\\/index.tsx → lib\\/index.js, lib\\/index.esm.js... [!] (plugin rpt2) RollupError: src\\/index.tsx:160:31 - error TS2354: This syntax requires an imported helper but module 'tslib' cannot be found.

160     const maxWidth = Math.max(...widthList);
                                  ~~~~~~~~~~~~
```

**Steps to Reproduce**  
1. Use Rollup to build a TypeScript project with the following syntax in `src/index.tsx`:
   ```javascript
   const maxWidth = Math.max(...widthList);
   ```
2. Run the Rollup build process.
   ```bash
   npm run dev  
   ```
3. Observe the error output indicating that `tslib` cannot be found.

**Expected Behavior**  
The Rollup build process should succeed without errors, and the bundled files should be generated.


**Proposed Fix**  
Ensure that `tslib` is installed and listed as a dependency in your project:
1. Install `tslib`:
   ```bash
   npm install tslib --save-dev
   ```
2. Verify that `tslib` is included in your `package.json` under development dependencies.

--- a/package.json
+++ b/package.json
@@ -47,6 +47,7 @@
     "rollup-plugin-postcss": "^4.0.2",
     "rollup-plugin-typescript2": "^0.34.1",
     "sass": "^1.58.3",
+    "tslib": "^2.8.1",
     "typescript": "^4.9.5"
   },
   "files": [

--- a/rollup.config.mjs
+++ b/rollup.config.mjs
@@ -7,8 +7,16 @@ import babel from '@rollup/plugin-babel';
 import terser from '@rollup/plugin-terser';
 import eslint from '@rollup/plugin-eslint';
 import banner2 from 'rollup-plugin-banner2';
+import fs from 'node:fs';
+import { dirname, join } from 'node:path';
+import { fileURLToPath } from 'node:url';
 
-import packageJson from './package.json' assert { type: 'json' };
+const configPath = join(
+  dirname(fileURLToPath(import.meta.url)),
+  './package.json'
+);
+const packageJson = JSON.parse(fs.readFileSync(configPath, 'utf8'));
+
 
 const extensions = ['js', 'jsx', 'ts', 'tsx', 'mjs'];
 

